src/components/backend-ai-resource-group-list.ts

```diff
@@ -848,6 +856,7 @@ export default class BackendAIResourceGroupList extends BackendAIPage {
                   validationMessage="${_t('data.explorer.ValueRequired')}"
                   required
                   autoValidate
+                  pattern="^[ㄱ-ㅎ가-힣a-zA-Z0-9]+(\\s[ㄱ-ㅎ가-힣a-zA-Z0-9]+)*$"
```

Comment:
And we need to consider unicode, not just for English and Korean. In Server-side, Resource Group name allows unicode, which contains special characters.

### Screenshot(s):

* WebUI:
![Screenshot 2024-12-04 at 5.23.14 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/Dgoz5XqHffJdivyccymr/746b2316-9488-4cb6-a7c6-cdd5b30efb2c.png)


* CLI:
![Screenshot 2024-12-04 at 5.25.51 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/Dgoz5XqHffJdivyccymr/41446708-6a0e-40ba-8f75-b4a076f1b4b6.png)

Reply:
Fixed to allow Unicode.